hr_employee_contract_history/tests/test_hr_employee_contract_history.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+###############################################################################
+#
+#    Copyright (C) 2015 Salton Massally (<smassally@idtlabs.sl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+from openerp.tests import common
+
+
+class TestEmployeeContractHistory(common.TransactionCase):
+
+    def setUp(self):
+        super(TestEmployeeContractHistory, self).setUp()
+        self.contract_model = self.env["hr.contract"]
+        self.employee_model = self.env['hr.employee']
+
+        # Create an employees
+        self.employee = self.employee_model.create({
+            'name': 'Employee 1',
+        })
+        self.contract_1 = self.contract_model.create(
+            {
+                'employee_id': self.employee.id,
+                'name': 'Contract 1',
+                'date_start': '2012-01-01',
+                'date_end': '2012-12-31',
+                'wage': 5000
+            }
+        )
+        self.contract_2 = self.contract_model.create(
+            {
+                'employee_id': self.employee.id,
+                'name': 'Contract 1',
```

Comment:
Rename the contract as Contract 2, in the code you added the job_id required, so probably it's usefull to add the job_id data in the test, probably to add a test if the job_id from the employee is changing when you change the job_id of the last contract...